src/dyad/dtl/CMakeLists.txt

```diff
@@ -25,6 +24,19 @@ set(DTL_PUBLIC_HEADERS ${DTL_PUBLIC_HEADERS} ${FLUX_PUBLIC_HEADERS})
 
 add_library(${PROJECT_NAME}_dtl SHARED ${DTL_SRC})
 target_link_libraries(${PROJECT_NAME}_dtl PUBLIC ${PROJECT_NAME}_utils Jansson::Jansson flux::core)
+#=======
+#elseif(DYAD_DATA_PLANE STREQUAL "FLUX_RPC")
+#    set(DTL_SRC ${DTL_SRC} ${FLUX_DTL_SRC})
+#    set(DTL_PRIVATE_HEADERS ${DTL_PRIVATE_HEADERS} ${FLUX_PRIVATE_HEADERS})
+#    set(DTL_PUBLIC_HEADERS ${DTL_PUBLIC_HEADERS} ${FLUX_PUBLIC_HEADERS})
+#else()
+#    message(FATAL_ERROR "-- [${PROJECT_NAME}] Unknown DYAD_DATA_PLANE configuration ${DYAD_DATA_PLANE}")
+#endif()
+
+#add_library(${PROJECT_NAME}_dtl SHARED ${DTL_SRC})
+#target_link_libraries(${PROJECT_NAME}_dtl PUBLIC utils Jansson::Jansson flux::core)
+#>>>>>>> fbec6e7 (refactored and initial changes to build.)
+target_link_libraries(${PROJECT_NAME}_dtl PUBLIC ${PROJECT_NAME}_utils Jansson::Jansson flux::core ${PROJECT_NAME}_perf)
```

Comment:
I was not sure how to resolve this conflict while rebasing. So, I left it as a comment block.